front/lib/api/assistant/agent.ts

```diff
@@ -128,6 +126,7 @@ export async function* runMultiActionsAgentLoop(
   // Citations references offset kept up to date across steps.
   let citationsRefsOffset = 0;
 
+  let processedContent = "";
```

Comment:
we accumulate the "processedContent". Once we get rid of `agentMessage.content`, I think we can decide to get rid of this entirely (and the front will no longer need to handle `agent_message_success`)